client/src/common/components/GeoLocation/index.tsx

```diff
@@ -46,7 +46,17 @@ const GeoLocation: FunctionComponent = () => {
     <S.GeoLocation>
       <Flex>
         <IconLocation />
-        {renderConditionalJSX(data, posLoading || infoLoading, posError || infoError)}
+        <RequestDataWrapper
+          data={data}
+          loading={posLoading || infoLoading}
+          error={posError || infoError}
+        >
+          {data && (
```

Comment:
no need to use condition `data && ...` here, you already have this check inside `RequestDataWrapper`

Reply:
I get and error `Uncaught TypeError: Cannot read properties of undefined (reading 'name')`, if I omit `data &&`.

Reply:
you can use this syntax - `<span>{data?.name}</span>`, please read about it

Reply:
Oh, I've actually used this syntax in this component already (`${pos?.lon}, ${pos?.lat}`). I guess I did not fully understand how it works. Will read more. Thank you